client/package.json

```diff
@@ -24,21 +29,23 @@
     "moment": "2.24.0",
     "prettier": "1.18.2",
     "react": "16.8.6",
+    "react-apollo": "2.5.8",
     "react-apollo-hooks": "0.5.0",
     "react-dom": "16.8.6",
     "react-router-dom": "5.0.1",
     "react-router-transition": "1.3.0",
     "react-scripts": "3.0.1",
     "styled-components": "4.3.2",
-    "typescript": "3.5.3"
+    "typescript": "3.6.0"
```

Comment:
Could you revert this change? I see the build failing as a version `3.6.0` could not be found.